endpoints/main.go

```diff
@@ -47,12 +58,16 @@ func getHTTPRequestTiming(urlString string, organizationName string, recordLongR
 }
 
 // Records information gathered from the capability statment into prometheus
-func recordLongRunningStats(resp *http.Response, organizationName string) {
-	var capabilityStatement = fhir.ParseCapabilityStatement(resp)
-	var fhirVersionString string = capabilityStatement.FhirVersion.Value
-	var fhirVersionAsNumber, _ = strconv.Atoi(strings.Replace(fhirVersionString, ".", "", -1))
-	fhirVersionGaugeVec.WithLabelValues(organizationName).Set(float64(fhirVersionAsNumber))
-	tlsVersionGaugeVec.WithLabelValues(organizationName).Set(float64(resp.TLS.Version))
+func recordLongRunningStats(resp *http.Response, organizationName string, urlString string) {
+	var capabilityStatement, err = fhir.ParseCapabilityStatement(resp)
+	if err != nil {
+		log.WithFields(log.Fields{"organization": organizationName, "url": urlString}).Warn("Capability Statement Response Parsing Error: ", err.Error())
```

Comment:
is the url string being passed in just for logging purposes?
does it make sense for the error to be returned and then logged in the parent function where the url string lives?
I could see it both ways. let me know your thoughts.

Reply:
I originally had it that way but I liked having the structure of if (error directly from parsing) log error ELSE (no error) record the parsed data. I _did_ try using resp.URL.host, but that does not return the full URL, only the hostname (aka www.google.com instead of www.google.com/fhir/dstu2/metadata)